File: src/client/java/fr/hardel/asset_editor/client/javafx/routes/changes/ChangesMainPage.java
package fr.hardel.asset_editor.client.javafx.routes.changes;

import fr.hardel.asset_editor.client.javafx.lib.Page;
import javafx.geometry.Pos;
import javafx.scene.control.Label;
import javafx.scene.layout.StackPane;
import net.minecraft.client.resources.language.I18n;

public final class ChangesMainPage extends StackPane implements Page {

    public ChangesMainPage() {
        getStyleClass().add("concept-main-page");
        setAlignment(Pos.CENTER);
        Label label = new Label(I18n.get("changes:layout.title"));
        label.getStyleClass().add("concept-main-placeholder");
        getChildren().add(label);
    }
}

File: src/client/kotlin/fr/hardel/asset_editor/client/compose/routes/changes/ChangesMainPage.kt
package fr.hardel.asset_editor.client.compose.routes.changes

import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import fr.hardel.asset_editor.client.compose.StudioColors
import fr.hardel.asset_editor.client.compose.StudioTypography
import net.minecraft.client.resources.language.I18n

@Composable
fun ChangesMainPage(modifier: Modifier = Modifier) {
    Box(
        contentAlignment = Alignment.Center,
        modifier = modifier.fillMaxSize()
    ) {
        Text(
            text = I18n.get("changes:layout.title"),
            style = StudioTypography.semiBold(24),
            color = StudioColors.Zinc400
        )
    }
}
